Sources/PaymentsCore/Networking/APIClientError.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+enum APIClientError {
+
+    static let domain = "APIClientErrorDomain"
+
+    enum Code: Int {
```

Comment:
qq: Can `Code` be made private here?

Reply:
We could! Question is whether or not we we think it will be helpful for merchants to see these codes?

Reply:
Ah right yeah that's a good point. In that case we'd want the docs to be public I agree.

Reply:
I mean the code is really meant for us mostly - but we can leave public in case merchants want that extra detail